common/rdm/RDMAPI.cpp

```diff
@@ -2556,6 +2556,128 @@ bool RDMAPI::SetDnsDomainName(
     error);
 }
 
+/*
+ * Fetch the dimmer curve
```

Comment:
Is this only for comment blocks with @params after them, or all comment blocks preceding a function call? I can take care of them in one go here.

Reply:
Generally if it's only one line we want to tag it as brief because then Doxygen can do some magic and show it in a few more places:
https://www.stack.nl/~dimitri/doxygen/manual/commands.html#cmdbrief

It would be all Doxygen style ones, so if they've already got @param or @return or whatever in there too.